### PR TITLE
doc: add blurb recommending the use of badges

### PR DESCRIPTION
**Pull request by gabrielschulhof**
Add blurb to N-API documentation indicating that we recommend the use
of badges.

Fixes: https://github.com/nodejs/node/issues/22796
Re: https://github.com/nodejs/node/pull/22801

This brings @ddlan's doc additions to the README.md in this repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,17 @@ In addition to running the tests in the converted modules we also have
 * Review the [roadmap](https://github.com/nodejs/abi-stable-node/milestones) and see how you
 can help accelerate this project.
 
+**Badges**
+
+The use of badges is recommended to indicate the minimum version of N-API
+required for the module. This helps to determine which Node.js major versions
+are supported. The following badges are available:
```
Comment (NickNaso) [inline README.md]: What do you think to link here the N-API version matrix https://nodejs.org/dist/latest/docs/api/n-api.html#n_api_n_api_version_matrix ?

**Review by NickNaso — APPROVED** [review]
LGTM

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by gabrielschulhof** [comment]
Landed in 0679f6514ab7822200319f3ad7e0265f659e1d1e.

**Pull request closed without merging**